observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc

```diff
@@ -25,10 +29,70 @@ include::modules/distr-tracing-tempo-install-web-console.adoc[leveloffset=+2]
 
 include::modules/distr-tracing-tempo-install-cli.adoc[leveloffset=+2]
 
+include::modules/distr-tracing-tempo-storage-ref.adoc[leveloffset=+1]
+
+include::modules/distr-tracing-tempo-object-storage-setup-aws-sts-install.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://docs.aws.amazon.com/iam/[AWS Identity and Access Management Documentation]
+* link:https://docs.aws.amazon.com/cli/[AWS Command Line Interface Documentation]
+* xref:../../../authentication/identity_providers/configuring-oidc-identity-provider.adoc#configuring-oidc-identity-provider[Configuring an OpenID Connect identity provider]
+* link:https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html[Identify AWS resources with Amazon Resource Names (ARNs)]
+
+include::modules/distr-tracing-tempo-object-storage-setup-ibm-storage.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[Getting started with the OpenShift CLI]
+* link:https://cloud.ibm.com/docs/cli?topic=cli-getting-started[Getting started with the {ibm-cloud-title} CLI] ({ibm-cloud-title} Docs)
+* link:https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-provision[Choosing a plan and creating an instance] ({ibm-cloud-title} Docs)
+* link:https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-getting-started-cloud-object-storage&q=credential&tags=cloud-object-storage&offset=10#getting-started[Getting started with {ibm-cloud-title} Object Storage: Before you begin] ({ibm-cloud-title} Docs)
+
+[id="configuring-permissions-and-tenants"]
+== Configuring the permissions and tenants
+
+Before installing a `TempoStack` or `TempoMonolithic` instance, you must define one or more tenants and configure their read and write access. You can configure such an authorization setup by using a ClusterRole and ClusterRoleBinding for the Kubernetes Role-Based Access Control (RBAC). By default, no users are granted read or write permissions. For more information, see "Configuring the read permissions for tenants" and "Configuring the write permissions for tenants".
+
+[NOTE]
+====
+The OpenTelemetry Collector of the {OTELName} can send trace data to a `TempoStack` or `TempoMonolithic` instance by using the service account with RBAC for writing the data.
+====
+
+[id="authentication_and_authorization_{context}"]
+.Authentication and authorization
+|===
+| Component | Tempo Gateway service | OpenShift OAuth | `TokenReview` API | `SubjectAccessReview` API
+
+|Authentication
+|&#10003;
+|&#10003;
+|&#10003;
+|
+
+|Authorization
+|&#10003;
+|
+|
+|&#10003;
```

Comment:
So I don't like this rule myself but I'm aware of it - in support matrix tables and stuff of that sort, [we're supposed to use plain X characters](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#support-matrix-tables) instead of anything else like checks or emojis for the sake of readability:
```suggestion
|X
|X
|X
|

|Authorization
|X
|
|
|X
```